packages/network-controller/src/types.ts

```diff
@@ -0,0 +1,5 @@
+import type { ProviderEngine } from 'web3-provider-engine';
+
+export type Provider = ProviderEngine;
```

Comment:
This used to be a custom type, but is now the same type that `web3-provider-engine` returns.